meta-mel/conf/local.conf.append.cyclone5

```diff
@@ -0,0 +1,3 @@
+PREFERRED_PROVIDER_virtual/bootloader = "u-boot"
+UBOOT_MACHINE = "socfpga_cyclone5_defconfig"
```

Comment:
Since this snippet gets appended to local.conf, where the user/customer will see it, I'd recommend adding one or more comments to this so they know why it's being done.